### DISCUSSION TITLE
S32K344 Can bootloader for autosar 4.7 sw version: 3.0.0

### DISCUSSION THREAD

**Discussion by yavuzklcc** [ecubus/EcuBus-Pro#51] [General]
Hi Zeng,
I am using the S32K344 CAN Bootloader. However, I noticed that the project is compatible with AUTOSAR 4.4.0. To run this project with AUTOSAR 4.7.0, I created a new project and compiled it accordingly. However, I am unable to upload the code. Aside from the MCAL libraries, do I need to make any changes to the code related to the S32K344? If I share the project with you, could you please help me?
Thanks..
Best regards.
you can contact to me this address :yavuzkilic1903@hotmail.com

**Comment by yavuzklcc** [comment]
ı get can bootloader from this link  :  https://community.nxp.com/t5/S32K-Knowledge-Base/Unified-bootloader-Demo/ta-p/1423099

**Comment by frankie-zeng** [comment]
> Hi Zeng, I am using the S32K344 CAN Bootloader. However, I noticed that the project is compatible with AUTOSAR 4.4.0. To run this project with AUTOSAR 4.7.0, I created a new project and compiled it accordingly. However, I am unable to upload the code. Aside from the MCAL libraries, do I need to make any changes to the code related to the S32K344? If I share the project with you, could you please help me? Thanks.. Best regards. you can contact to me this address :[yavuzkilic1903@hotmail.com](mailto:yavuzkilic1903@hotmail.com)

Hi, guy, you can contact nxp FAE.
